docs/build/configure/triggers.md

```diff
@@ -178,8 +221,8 @@ If you prefer to configure your trigger with raw JSON instead of the config buil
 {{% /tab %}}
 {{< /tabs >}}
 
```

Comment:
I think we need another table here explaining the attributes, whether they are required and their description; similar to what we do for component models: https://docs.viam.com/components/arm/xarm6/ 

I don't for example know what I'd put for `Min-Time-Received`, what format to use, or whether it's even required